dirs/dirs.go

```diff
@@ -126,9 +128,11 @@ func SetRootDir(rootdir string) {
 
 	switch release.ReleaseInfo.ID {
 	case "fedora", "centos", "rhel", "arch":
-		SnapMountDir = filepath.Join(rootdir, "/var/lib/snapd/snap")
+		BaseSnapMountDir = "/var/lib/snapd/snap"
```

Comment:
`HostSnapMountDir` still has a notion of not saying that it is absolute or not. `BaseSnapMountDir` was my attempt to word that. What about `RawSnapMountDir`?